app/services/record_declarations/base.rb

```diff
@@ -65,13 +64,8 @@ def create_declaration_attempt!
     end
 
     def find_or_create_record!
-      ApplicationRecord.transaction do
-        self.class.declaration_model.find_or_create_by!(declaration_parameters) do |participant_declaration|
-          DeclarationState.create!(
-            participant_declaration: participant_declaration,
-            state: user_profile.fundable? ? "eligible" : "submitted",
-          )
-        end
+      self.class.declaration_model.find_or_create_by!(declaration_parameters).tap do |participant_declaration|
+        participant_declaration.make_eligible! if fundable?
```

Comment:
The old code created `submitted` state. Does the new code do that too?

Reply:
It's the default and now creates submitted first and then eligible, so it does have a definite first state.